Makefile

```diff
@@ -22,6 +22,7 @@ upgrade-piptools: piptools # upgrade pip-tools using pip-tools.
 
 upgrade: export CUSTOM_COMPILE_COMMAND=make upgrade
 upgrade: upgrade-piptools piptools ## upgrade requirement pins.
+	pip-compile --allow-unsafe --rebuild -o requirements/pip.txt requirements/pip.in
 	pip-compile requirements/base.in --rebuild --upgrade -o requirements/base.txt
 	pip-compile requirements/test.in --rebuild --upgrade -o requirements/test.txt
 	pip-compile requirements/docs.in --rebuild --upgrade -o requirements/docs.txt
```

Comment:
We should only need `--rebuild` on the first pip-compile run in the upgrade target.